.gitignore

```diff
@@ -15,6 +15,7 @@ coverage
 
 # Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
 .grunt
+.idea
```

Comment:
This should not be in a project gitignore. If you want git to ignore it, put it in the global config.